www/include/common/common-Func.php

```diff
@@ -657,13 +409,15 @@ function getMyServiceMacro($service_id, $field)
         return;
     }
     global $pearDB, $oreon;
```

Comment:
```suggestion
    global $pearDB,;
```
$oreon is never used